flask_restplus_patched/parameters.py

```diff
@@ -72,3 +76,23 @@ def __init__(self, *args, **kwargs):
             self.fields['op'].validators + [validate.OneOf(self.OPERATION_CHOICES)]
         self.fields['path'].validators = \
             self.fields['path'].validators + [validate.OneOf(self.PATH_CHOICES)]
+
+    @validates_schema
+    def validate_patch_structure(self, data):
+        """
+        Common validation of PATCH structure
+
+        Provide check that 'value' present in all operations expect it.
+
+        Provide check if 'path' is present. 'path' can be absent if provided
+        without '/' at the start. Supposed that if 'path' is present than it
+        is prepended with '/'.
+        Removing '/' in the beginning to simplify usage in resource.
+        """
+        if data['op'] not in self.NO_VALUE_OPERATIONS and 'value' not in data:
+            raise ValidationError('value is required')
+
+        if 'path' not in data:
```

Comment:
Nope, I meant that `path` *may be* **always** present by the time you reach this line of code because `path = base_fields.String(required=True)` (notice `required=True`).